packages/x-codemod/src/v8.0.0/charts/rename-sparkline-colors-to-color/expected.spec.tsx

```diff
@@ -0,0 +1,19 @@
+/* eslint-disable no-restricted-imports */
+import * as React from 'react';
+import { SparkLineChart } from '@mui/x-charts';
+
+const data = [1, 2];
+
+function Chart() {
+  const fn = (mode) => (mode === 'light' ? ['black'] : ['white']);
+
+  // prettier-ignore
+  return (
+    (<React.Fragment>
+      <SparkLineChart data={data} color={['red']?.[0]} />
```

Comment:
I followed a "better safe than sorry" approach here. Adding the optional chain operator might save us from some crashes in edge cases. Happy to remove it if you think that's too much.

Reply:
what will happen to `var` when `var=['blue']`?